assignment/assignment-8-timed/assignment-8.2.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.2</title>
+    </head>
+    <body>
+        <form>
+            <h1>Simple Quiz</h1>
+            <div>
+                <label for="fullNameInput">Full Name</label>
+                <input type="text" name="fullName" id="fullNameInput" />
+
+                <label for="emailInput">Email</label>
+                <input type="email" name="email" id="emailInput" />
```

Comment:
would be better if you add placeholder attr for full name and email inputs

Reply:
sounds good. thanks